glean-core/python/glean/glean.py

```diff
@@ -230,6 +230,9 @@ def _initialize_with_tempdir_for_testing(
         The temporary directory will be destroyed when Glean is initialized
         again or at process shutdown.
         """
+        # Lower throttle backoff time for testing
+        PingUploadWorker._throttle_backoff_time = 1
```

Comment:
Why is Python the only one where we lower this value?

Reply:
Because it is the only one that takes a long time to run the test suite with this change.